### ISSUE TITLE
color code event categories

### ISSUE THREAD

**Issue by bdolor** [BCcampus/eypd#379]
### Description 

Currently event categories have the ability to add a color value. Depending on that value, we need to make the listing of the categories on the myEYPD page the same colour.

### Expected behavoiur

Category text needs to use the colour value that's saved with the event category.
![image](https://user-images.githubusercontent.com/2048170/38822850-8b6f1520-4159-11e8-9096-0ea6a58dd4c9.png)

### Actual Behaviour

All text is treated equally (black)

### Steps to reproduce the problem

go to myEYPD page

**Comment by alex-418** [comment]
Front half completed via #402

**Comment by paulagaube** [comment]
@mandily-p @kathreenriel @brettjforsyth 
I took a stab at changing the colours in Admin > myEYPD > Event Categories
Chose HEX colors from a palette generated at: http://colorsafe.co/

Tested the colour display on various user profiles and the colours appear to be working properly in the donut. (Whether we want to tweak the colours is another matter. At least they are not mostly bright green and white now. ) 

Hex Colour / Category
#005051 Administration and Communication 
#800000 Child Growth and Development 
#DC1829 Child Health and Wellbeing 
#34515e Curriculum/Program Development 
#9932cc Early Care and Education 
#3455db Extra Support 
#1d781d Family Support 
#008080 Indigenous 
#007faa Kindergarten Transitions 
#34515e Middle Years 
#b22222 Online 
#8d6708 Professional Ethics & Practice 
#aa0000 Professional Networking

**Comment by kathreenriel** [comment]
Colour palette looks good to me too. @mandily-p your thoughts?

**Comment by mandily-p** [comment]
For reference I put the colours Paula chose into Illustrator so I could see them together as a whole.
![screen shot 2018-11-28 at 1 08 51 pm](https://user-images.githubusercontent.com/20425828/49182398-0e03f700-f30f-11e8-9952-869b14dac013.png)


I have two thoughts:
1. Some of these colours are too close to each other to be easily recognizable, and in some cases may even blend into each other in the wheel. (Middle years and curriculum development is actually the same colour)
2. I'd like to group types of categories together by colour.

I'll through this back into to do with my name on it.

**Comment by mandily-p** [comment]
@kathreenriel Can you validate that I've grouped these categories appropriately. Feel free to edit my comment here to straighten things out if needed.

Child growth and development
Child health and wellbeing
Middle years

Early care and education
Extra support
Family support
Kindergarten transitions

Administration and communication
Curriculum/program development
Professional networking
Professional ethics and practice

Indigenous

Online (should this even be a category? I'd rather make it a location)

**Comment by brettjforsyth** [comment]
@kathreenriel @mandily-p at this point can we start a new issue for the colors as this isn't a coding related task anymore?

**Comment by kathreenriel** [comment]
@brettjforsyth I just added a non-dev tag to clarify

**Comment by kathreenriel** [comment]
@mandily-p let's keep Online as a category so that users can search for just online events - for now.

**Comment by kathreenriel** [comment]
@mandily-p changed the order of some of the categories above

**Comment by mandily-p** [comment]
@kathreenriel If you're ok with this colour palette, I'll go ahead and populate in Wordpress. I've attached a pdf that includes all variations of colour-blindness. [color-blind-palette.pdf](https://github.com/BCcampus/eypd/files/2636726/color-blind-palette.pdf)

**Comment by mandily-p** [comment]
How about this @kathreenriel ?

<img width="736" alt="screen shot 2018-12-04 at 8 30 46 am" src="https://user-images.githubusercontent.com/20425828/49457103-19d43b00-f79f-11e8-969e-3318e16338e0.png">

**Comment by kathreenriel** [comment]
@mandily-p It looks good - Admittedly I'm not sure until I see it in the donut @paulagaube your thoughts?

**Comment by paulagaube** [comment]
@mandily-p @kathreenriel The latest palette looks good to me, too. Amanda, could you add the HEX values next to the colour palette so that we have a record of that for future? That will make it easier to update WordPress as well.

**Comment by mandily-p** [comment]
<img width="490" alt="screen shot 2018-12-04 at 1 50 57 pm" src="https://user-images.githubusercontent.com/20425828/49475346-d47a3280-f7cb-11e8-84cf-bf4fbb6ff4c8.png">

**Comment by mandily-p** [comment]
From my account on cert. 

@kathreenriel Should we look at ordering categories in their genres as I did in the mock or are we ok with the order of them now?
<img width="744" alt="screen shot 2018-12-04 at 2 08 19 pm" src="https://user-images.githubusercontent.com/20425828/49476224-3b98e680-f7ce-11e8-804f-0c477935d8c5.png">

**Comment by mandily-p** [comment]
The order of the colours in the donut should be the same order in the legend. @brettjforsyth Can you tell us where we can edit these please?

**Comment by kathreenriel** [comment]
Yes it would be ideal if the colours in the legend were in order with the colours as they are represented in the donut. 
<img width="616" alt="screen shot 2018-12-04 at 2 55 47 pm" src="https://user-images.githubusercontent.com/16693738/49478659-3e4b0a00-f7d5-11e8-9794-f9848d0fbda7.png">

**Comment by kathreenriel** [comment]
@mandily-p does the colour scheme order in the graphic above reflect your idea of how the colours should render in the legend?

**Comment by brettjforsyth** [comment]
@kathreenriel I would like to move this to validating or done as the original ticket discusses adding the colors to the donut. I have captured this new work (after discussing with @bdolor) in a new issue.

**Comment by kathreenriel** [comment]
@paulagaube Extra Support and Special Needs mean the same thing I think we should remove rename extra support to special needs on cert

**Comment by paulagaube** [comment]
@kathreenriel I have renamed Extra Support on CERT and now the categories match on CERT and PROD.

**Comment by paulagaube** [comment]
I am updating the colours on PROD to match CERT.

**Comment by paulagaube** [comment]
This is DONE

**Comment by kathreenriel** [comment]
@paulagaube Thank you
